validate.go

```diff
@@ -208,6 +211,23 @@ func checkLinux(spec rspec.Spec) {
 			logrus.Fatalf("namespace %v is invalid.", spec.Linux.Namespaces[index])
 		} else if spec.Linux.Namespaces[index].Type == rspec.UTSNamespace {
 			utsExists = true
```

Comment:
I think we want:

    else if len(spec.Linux.Namespaces[index].Path) == 0 {
        continue
    }

as the first block after the validity check.  That way [the logic][1] also applies to this UTS case, and we don't have to duplicate `Path` checks for each namespace below.

[1]: https://github.com/opencontainers/ocitools/pull/124#discussion_r68693222